src/allencell_ml_segmenter/prediction/model.py

```diff
@@ -4,32 +4,16 @@
 
 from allencell_ml_segmenter.core.event import Event
 from allencell_ml_segmenter.core.publisher import Publisher
+from allencell_ml_segmenter.core.file_input_model import FileInputModel
 
 
-class PredictionInputMode(Enum):
-    FROM_PATH = "from_path"
-    FROM_NAPARI_LAYERS = "from_napari_layers"
-
-
-class PredictionModel(Publisher):
+class PredictionModel(FileInputModel):
```

Comment:
What do you think about using model composition instead of inheritance? To me it seems less like `PredictionModel` is a `FileInputModel` and more like `PredictionModel` has a `FileInputModel` because it has a file input widget as part of its UI.

Reply:
I think this is probably a good idea, inline with stuff i have read in recent years.